<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Biography</class>
 <widget class="QWidget" name="Biography">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>771</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btn_info">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>121</width>
     <height>44</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Informaci√≥n</string>
   </property>
   <property name="icon">
    <iconset theme="ssdf">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btn_publi">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>131</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Publicaciones</string>
   </property>
   <property name="icon">
    <iconset theme="ssdf">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btn_segui">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>111</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Calibri&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Seguidores</string>
   </property>
   <property name="icon">
    <iconset theme="ssdf">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QStackedWidget" name="Bio_windows">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
